docs/quickstart.rst

```diff
@@ -14,6 +14,15 @@ following::
     make db_seed_basic
     make devserver
 
+.. collapse:: Debug info
+
+    In case some python requirement is missing during the above, install it as follows:: 
+    
+        bash
+        source env/bin/activate
+        pip install PACKAGE_NAME
```

Comment:
I think it would be better to run `pip install -r requirements.txt` so we get the right versions too

Reply:
actually `make install` should've taken care of it. Abandoning this in favor of another tip.